app/views/comments/create.html.erb

```diff
@@ -0,0 +1,12 @@
+<!-- Creating error message displaying the data that failed -->
+<% if @comment.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@comment.errors.count, "error") %> prohibited this comment from being saved:</h2>
+
+    <ul>
+    <% @comment.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
```

Comment:
- [OPTIONAL] You have done a great job for creating a form to add comments ✨💪, However, I think it would be better to have a form where the current user can add comments to a post in `views/comments/new.html.erb` file which accepts input for creating comments. I noticed that you have a form in `views/posts/show.html.erb` However, it is violating the principles of separation of concerns and modularity in web development 📌⛏ Each component of your application (models, views, controllers) should have a clear and distinct responsibility.

    - _The `posts/show.html.erb` view is meant for displaying information about a single post, not for creating new posts/comments._
    - _The `comments/new.html.erb` view, on the other hand, should be responsible for rendering the form to create a new comment._

  Mixing these responsibilities can make the codebase less maintainable and harder to understand✨ So, please  make sure to keep the comment creation form in its own dedicated view (`views/comments/new.html.erb`)